app/packages/components/src/components/Loading/LoadingDots.module.css

```diff
@@ -0,0 +1,23 @@
+.loading {
+  display: inline-block;
+  height: 12px;
+  line-height: 12px;
+  overflow: hidden;
+  margin-left: 1px;
+  margin-right: 2px;
+}
+.loading::after {
+  display: inline-table;
+  white-space: pre;
+  content: "\A.\A..\A...";
+  animation: spin 1s steps(4) infinite;
+  margin-left: 1px;
+  margin-right: 2px;
+  letter-spacing: 0.1em;
```

Comment:
FYI- I used letter spacing to tweak the spacing of the dots a bit

Reply:
Thanks @brimoor

Reply:
Just updated the text color

https://user-images.githubusercontent.com/17770824/205979207-d359c95f-fb4b-4100-993d-639d2f0619ff.mov